cardano-crypto-class/cardano-crypto-class.cabal

```diff
@@ -38,6 +39,7 @@ common project-config
 library
   import:            base, project-config
   hs-source-dirs:    src
+  c-sources: cbits/blst_util.c
```

Comment:
This has already been added. See line 127 https://github.com/input-output-hk/cardano-base/blob/f855e4b571de47865341590196b712c7e20cf0d0/cardano-crypto-class/cardano-crypto-class.cabal#L127
```suggestion
```